Vagrantfile

```diff
@@ -6,17 +6,24 @@ NODES = 4
 DISKS = 8
 
 Vagrant.configure("2") do |config|
-    config.vm.box = "chef/centos-7.1"
```

Comment:
I am not familiar with ruby. I am managing with just enough syntax learnt from neighbouring code snippets. I have passed an ``override`` parameter to the provider block as suggested in http://docs.vagrantup.com/v2/providers/configuration.html. I tried the above suggested change only to find Vagrant complain,
 ``vm:
* A box must be specified.``.
I don't know how to get past this.